### PR TITLE
poretools 0.6.0

### PR DESCRIPTION
**Pull request by ilovezfs**
Created with `brew bump-formula-pr`.

### CODE REVIEW

File: poretools.rb [hunk 1]
```diff
@@ -87,51 +88,40 @@ class Poretools < Formula
     sha256 "76b00286acba1f65c76a3869bc60e099190ce48d0a5822606ce222e80529e523"
   end
 
+  def resources
+    front_load = [resource("six"), resource("numpy")]
+    front_load + super - front_load - [resource("Cython"), resource("test")]
```
Comment (ilovezfs) [inline poretools.rb]: @tdsmith heresy?

**Comment by ilovezfs** [comment]
@sjackman :penguin:

File: poretools.rb (hunk 1 above)
Comment (tdsmith) [inline poretools.rb]: weird but yolo

Comment (ilovezfs) [inline poretools.rb]: hehe. yeah. saves a bit of boilerplate and avoids converting from resource array to string array to set back to resource array, but a dsl for resource grouping/ordering might be good.

**Pull request closed without merging**